### ISSUE TITLE
How to build on Ubuntu Linux?

### ISSUE THREAD

**Issue by macOS26** [mistydemeo/cue2ccd#7]
I am new to Rust. 

What are the dependencies needed for Ubuntu?
What are the build steps for Ubuntu?

You should add these instructions to your ReadMe or have a how to compile/build section for Linux, Mac and Windows.

**Comment by mistydemeo** [comment]
Sorry, I missed this issue earlier!

You need to have `libcue` installed to be able to build this. You can install it on Ubuntu via `sudo apt-get install libcue-dev`. Once you've done that, `cargo build` should work.

I've just finished the features I needed for the 1.0 release, so I'll also be getting some prebuilt binaries together for Mac, Linux and Windows.

**Comment by mistydemeo** [comment]
I've just merged a change earlier today that means you don't need libcue installed anymore. On Linux and Mac, just running `cargo build` should work. Windows users still need to install flex and bison first, for example by running `choco install winflexbison`. I've updated the README with build instructions.

I'll have prebuilt binaries available soon, and those won't require any special dependencies.
